modules/@shopify/checkout-sheet-kit/src/index.ts

```diff
@@ -151,6 +216,20 @@ export class WebPixelsParseError extends Error {
   }
 }
 
+export class LifecycleEventParseError extends Error {
+  constructor(
+    message?: string | undefined,
+    options?: ErrorOptions | undefined,
```

Comment:
`undefined` is not necessary. If a parameter is optional, it is implicitly undefined when not provided.

Reply:
Yeah true